compile-release-tools

```diff
@@ -18,7 +18,16 @@ set -e
 set -u
 set -o pipefail
 
+RELEASE_TOOLS=(
+  blocking-testgrid-tests
+  krel
```

Comment:
Hah, this kinda clashes with the changes in https://github.com/kubernetes/release/pull/1046

I'm happy to bring my changes on top of that PR.

Reply:
Heh. Saw some of the convo in https://github.com/kubernetes/release/pull/1044 and figured this would be the quick fix. :stuck_out_tongue_winking_eye: